src/components/Landing/Landing.tsx

```diff
@@ -73,23 +72,8 @@ const LandingPage = () => {
         </Button>
       </Box>
 
-      {/* Login Modal */}
-      <Modal open={open} onClose={handleClose}>
-        <Box
-          sx={{
-            position: 'absolute',
-            top: '50%',
-            left: '50%',
-            transform: 'translate(-50%, -50%)',
-            bgcolor: 'background.paper',
-            p: 4,
-            boxShadow: 24,
-            borderRadius: 2,
-          }}
-        >
-          <Login onClose={handleClose} />
-        </Box>
-      </Modal>
+      {/* Sign-in / Sign-up Modal */}
+      {open && <Login onClose={handleClose} open={open} />}
```

Comment:
❗ this does not make much sense since the component has a prop to which it is indicated whether it is open or not. Conditioning would be to mount and dismount the component based on the `open` state value.

Reply:
The bool condition is redundant here pal....